tests/integration/config/plugins/integration-tests/data/tests/tests_common.txt

```diff
@@ -214,9 +217,12 @@ test-data "Fighters and Carriers and Escorts Save"
 				zoom 1
 				angle 0
 				under
-			gun -7 -14 "Meteor Missile Launcher"
+			gun -7 -14 Flamethrower
```

Comment:
It wasn't listed as "Flamethrower" in the original test (and isn't in the outfit list either), so I think I would prefer to just leave it as Flamethrower. It theoretically shouldn't make a difference, but I believe the test is supposed to represent how the things are saved by the game, thus replicating reading things properly. So the Flamethrower entry there should remain in whatever state the game saves it in the save file.

Reply:
The game saves single-word tokens without quotation marks.
Actually makes me wonder why the data files put quotation marks around them in the first place...

Reply:
OK, just checked in game (bought some flamethrowers and installed them on my scout, took off and landed and then exited and checked the save file to see how they are stored. 

```
	gun -18.5 -6 Flamethrower
		under
	gun 18.5 -6 Flamethrower
		under
```

As such, I'm going to leave them without the quotation marks in the test.

Reply:
> The game saves single-word tokens without quotation marks. Actually makes me wonder why the data files put quotation marks around them in the first place...

As best I can recall, it is exclusively because some people wanted to standardize the data files.